charts/chirpstack/templates/chirpstack/ingress.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.ingress.enabled -}}
+{{- if false }}
```

Comment:
What exactly is checked here? :sweat_smile:

Reply:
This is for the IDE, in our case IntelliJ, so that it knows what to autocomplete